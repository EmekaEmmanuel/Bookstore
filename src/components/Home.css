* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
  font-family: 'Open Sans', sans-serif;
}

.home_section {
  padding: 100px 0;
  width: 90%;
  margin: auto;
  letter-spacing: 0.8px;
}

.home_section header {
  font-size: 35px;
  font-weight: 500;
}

.home_section p {
  font-size: 30px;
  font-weight: 500;
  margin: 20px 0;
  line-height: 40px;
  padding: 20px 0 10px 0;
}
