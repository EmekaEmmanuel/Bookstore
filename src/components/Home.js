import React from 'react';
import Nav from './Nav';
import './Home.css';

function Home() {
  /* eslint-disable max-len */

  return (
    <div>
      <Nav />

      <section className="home_section">

        <header className="home_header">
          Welcome to our page!
        </header>

        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum quia impedit rerum. Similique sapiente quibusdam esse voluptatibus fugiat. Animi eum quisquam quo, vitae neque voluptates iusto placeat accusantium deserunt nihil. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quaerat exercitationem ad modi similique corporis aperiam. Eligendi hic architecto perspiciatis dignissimos totam voluptates asperiores, repudiandae culpa ad sunt quas cupiditate magnam. </p>
        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quaerat exercitationem ad modi similique corporis aperiam. Eligendi hic architecto perspiciatis dignissimos totam voluptates asperiores, repudiandae culpa ad sunt quas cupiditate magnam.</p>
      </section>
    </div>

  );
  /* eslint-enable max-len */
}

export default Home;
