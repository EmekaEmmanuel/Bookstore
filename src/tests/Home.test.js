import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('renders correctly', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <Home />
    </MemoryRouter>,
  )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
