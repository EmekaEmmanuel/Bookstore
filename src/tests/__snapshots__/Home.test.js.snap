// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="nav_container"
  >
    <header
      className="nav_header"
    >
      <h1>
        Math Magicians
      </h1>
      <nav
        className="nav_list"
      >
        <li>
          <a
            className="nav_anchor td_none"
            href="/"
            onClick={[Function]}
          >
            Home 
          </a>
        </li>
        <li
          className="border_active"
        >
          <a
            className="nav_anchor  td_none"
            href="/Calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            className="nav_anchor td_none"
            href="/Quote"
            onClick={[Function]}
          >
            Quote
          </a>
        </li>
      </nav>
    </header>
  </div>
  <section
    className="home_section"
  >
    <header
      className="home_header"
    >
      Welcome to our page!
    </header>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum quia impedit rerum. Similique sapiente quibusdam esse voluptatibus fugiat. Animi eum quisquam quo, vitae neque voluptates iusto placeat accusantium deserunt nihil. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quaerat exercitationem ad modi similique corporis aperiam. Eligendi hic architecto perspiciatis dignissimos totam voluptates asperiores, repudiandae culpa ad sunt quas cupiditate magnam. 
    </p>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quaerat exercitationem ad modi similique corporis aperiam. Eligendi hic architecto perspiciatis dignissimos totam voluptates asperiores, repudiandae culpa ad sunt quas cupiditate magnam.
    </p>
  </section>
</div>
`;
